src/gui/shareusergroupwidget.h

```diff
@@ -73,7 +73,7 @@ private slots:
     void slotSharesFetched(const QList<QSharedPointer<Share>> &shares);
 
     void on_shareeLineEdit_textChanged(const QString &text);
-    void searchForSharees(bool searchGlobally);
+    void searchForSharees(int lookupMode);
```

Comment:
Why is this an int and not the enum type? Would avoid the cast and you pass the enum in the call anyway.

Reply:
@er-vin If we want to have it, we have 2 ways:
- #include "sharee.h" in the "shareusergroupwidget.h" - at the moment ShareeModel is forward-declared, so we can't use enums from it
- moving enum to a separate header file "shareelokupmode.h" and including it in both "sharee.h" and "shareusergroupwidget.h"

Which way is preferable?